### PR TITLE
UI Layout Refactor

### PR DESCRIPTION
**Pull request by mhawryluk**

### CODE REVIEW

**Review by t0maboro — APPROVED** [review]
Everything looks good, I'm approving merging this

File: src/app.py [hunk 1]
```diff
@@ -1,12 +1,12 @@
 import sys
 from PyQt5.QtWidgets import QApplication
 
-from widgets.main_layout import RandomGenerator
+from widgets.main_layout import MainWindow
```
Comment (gregori0o) [inline src/app.py]:
```suggestion
from widgets import MainWindow
```

File: src/widgets/main_layout.py [hunk 2]
```diff
@@ -1,32 +1,28 @@
 from PyQt5.QtWidgets import QMainWindow, QHBoxLayout, QWidget
+from widgets import WINDOW_WIDTH, WINDOW_HEIGHT
+from .main_widget import MainWidget
```
Comment (gregori0o) [inline src/widgets/main_layout.py]:
```suggestion
from widgets import MainWidget
```

File: src/widgets/__init__.py [hunk 3]
```diff
@@ -0,0 +1,7 @@
+from .config import WINDOW_HEIGHT, WINDOW_WIDTH, UNFOLD_BUTTON_WIDTH, UNFOLD_WIDGET_WIDTH
+from .unfold_widget import UnfoldWidget
+from .algorithm_widget import AlgorithmWidget
+from .import_widget import ImportWidget
+from .preprocessing_widget import PreprocessingWidget
+from .visualization_widget import VisualizationWidget
+from .main_layout import MainWindow
```
Comment (gregori0o) [inline src/widgets/__init__.py:8]:
```suggestion
from .main_widget import MainWidget
from .main_layout import MainWindow
```

**Review by gregori0o — COMMENTED** [review]
I suggest only changing some imports to have one style of this.
But anyway good job. 🚀 🚀 🚀

**Comment by mhawryluk** [comment]
>  I suggest only changing some imports to have one style of this. But anyway good job. 🚀 🚀 🚀

Done!

**Pull request merged**